lib/model/special_triggers.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+import 'card/special/special_cards.dart';
+import 'game_state.dart';
+import 'hand.dart';
+
+void addSpecialTopCardToDeck(GameState state){
+
+  if (checkMutiny(state.playerHand)){
+    state.deck.insert(0, Mutiny());
+  }else if (checkNavyRaid(state)){
+    state.deck.insert(0, NavyRaid());
+  }else if (checkRavenousCrew(state.playerHand)){
+    state.deck.insert(0, RavenousCrew());
```

Comment:
Formatting has gone a bit wonky here - there's a command called `dartfmt` you can run on files to auto-format them for you (I get it as a right-click option in IntelliJ)

Reply:
Thanks for the tip, yes that has changed a bit of formatting.